typhon/utils.py

```diff
@@ -53,7 +56,7 @@ def is_signal_ro(signal):
 
 
 def grab_kind(device, kind):
-    """Grab all signals of a specific Kind from a Device"""
+    """Grab all signals of a specific Kind from a Device instance"""
```

Comment:
Mind just putting a docstring specifying the returned value? Should have been there before but I figured if I documented them poorly I'd struggle less breaking the API 😄 This seems generally useful outside `typhon`. I like having the `Component` available as well.